README.md

```diff
@@ -0,0 +1,58 @@
+# Rammingen
+
+Rammingen is a self-hosted backup encryption system.
+
+1. Syncs backed-up directories across several hosts.
+1. Manages previous versions of backups.
+
+Rammingen installation dependency graph (arrows represent the "X depends on Y" relationship):
+
+```mermaid
+graph BT
+    subgraph Server host
+        server[Rammingen server]
+        db[("Database<br>(Postgres)")]
+        storage[("Backup archive storage<br>(in the file system)")]
+    end
+
+    subgraph Client host
+        client["Rammingen client<br>(encrypts/decrypts backup)"]
+        directory[(Backed-up directory)]
+    end
+
+    server --> db
+    server --> storage
+    client --> server
+    client --> directory
+```
+
+## Quick start
+
+This guide will help you quickly deploy and try out Rammingen on your local machine.
```

Comment:
I think it's more important to suggest a configuration that's more realistic so that users get a better idea of the proper usage and configuration capabilities. 

rammingen-server is meant to be run on the server, not locally, so it's better to make it clear from the start. I would describe the following installation steps:
- Install nginx, postgres and docker on the server from system repository.
- Set up a postgres user and database.
- Create a storage directory.
- Create server configuration file.
- Initialize the database and user credentials with `rammingen-admin`.
- Run the server via a docker command.
- Set up nginx configuration and TLS.
- Create a local directory.
- Create an encryption key.
- Create client configuration file.
- Run the client via a docker command.

The following further steps can be recommended to improve the setup:
- Set up a systemd service for the server.
- Set up a systemd user service or cron for the client.

I can add this detailed description myself later, so it doesn't have to be done in this PR.

I guess the quick start guide can also be suggested as an alternative, but I'm not sure it's very useful: there is no easy way to migrate from that setup to a proper persistent setup, and there is no easy way to use file sync (I guess we could set up multiple clients for this). Maybe it will be more useful to replace it with an overview of the client commands so that the users get a good idea of the system's capabilities without needing to set up everything.